src/main/java/org/alliancegenome/curation_api/model/entities/AffectedGenomicModel.java

```diff
@@ -105,16 +105,4 @@ public class AffectedGenomicModel extends GenomicEntity {
 	@JsonView({View.FieldsAndLists.class, View.AffectedGenomicModelDetailView.class})
 	private List<AgmAgmAssociation> agmAgmAssociations;
```

Comment:
Should be called `parentalPopulations`

Reply:
and the above @`IndexedEmbedded` fields should be for the `agmAgmAssociationObject` not the subject - the subject is just the AGM that these associations are attached to so we don't need to index those fields